k8s/envoy-jwt-opa/scripts/set-env.sh

```diff
@@ -37,14 +37,14 @@ wait_for_envoy() {
     kubectl rollout status deployment/backend --timeout=60s
 
     LOGLINE="all dependencies initialized. starting workers"
-    LOGLINE2="DNS hosts have changed for backend-envoy"
+    LOGLINE2="membership update for TLS cluster backend added 1 removed 1"
```

Comment:
Sometimes Andrés says that the log messages can't be changed. So I'm not sure if this minor change can be made.

```suggestion
    LOGLINE2="Membership update for TLS cluster backend: added 1, removed 1"
```

Reply:
I worked side by side with Andres on this, and we found that the best way to check if Envoy is ready was changing the log message that is searched (using `grep`, lines below). This way, both OPA tests (_envoy-opa_ and _envoy-jwt-opa_) are consistent on which line they wait to appear in the logs.  
For this reason (`grep` command over Envoy logs used for checking readiness), logline must be exactly the string that it is, or it will be not found in the logs, thus test will fail.